src/extension-runners/firefox-android.js

```diff
@@ -42,6 +42,19 @@ import type {
 
 const log = createLogger(__filename);
 
+const ignoredParams = {
+  profilePath: '--profile-path',
+  keepProfileChanges: '--keep-profile-changes',
+  browserConsole: '--browser-console',
+  preInstall: '--pre-install',
+  startUrl: '--start-url',
+  binaryArgs: '--binary-args',
+};
+
+const getIgnoredParamsWarningsMessage = (optionName) => {
+  return `Firefox for Android target does not support ${optionName} option`;
```

Comment:
While you're at it, let's improve the English:

`The Firefox for Android target does not support the ${optionName} option`
or
`The Firefox for Android target does not support ${optionName}`.